### PR TITLE
UpdateRowsDeserializer.deserializeRow fails with EventDataDeserializationException No TableMapEventData has been found for table id

### PR DESCRIPTION
**Pull request by dartartem**
Fixed exception on cdc Restart.
See: https://github.com/eventuate-foundation/eventuate-cdc/issues/89#issuecomment-732082764

Fixed initial exceptions.
See: https://github.com/eventuate-foundation/eventuate-cdc/issues/89#issuecomment-732227158

### CODE REVIEW

File: eventuate-local-java-cdc-connector-db-log-common/src/main/java/io/eventuate/local/db/log/common/DatabaseOffsetKafkaStore.java [hunk 1]
```diff
@@ -42,6 +42,8 @@ public synchronized void save(BinlogFileOffset binlogFileOffset) {
       );
 
     CompletableFutureUtil.get(future);
+
+    logger.info("Offset is saved: {}", binlogFileOffset);
```
Comment (cer) [inline eventuate-local-java-cdc-connector-db-log-common/src/main/java/io/eventuate/local/db/log/common/DatabaseOffsetKafkaStore.java:46]: This should be debug logging

File: eventuate-local-java-cdc-connector-mysql-binlog/src/main/java/io/eventuate/local/mysql/binlog/MySqlBinaryLogClient.java [hunk 2]
```diff
@@ -201,6 +201,8 @@ private void handleRestart(Exception e) {
 
   private void handleBinlogEvent(Event event, Optional<BinlogFileOffset> binlogFileOffset) {
 
+    logger.info("Event received: {}", event);
```
Comment (cer) [inline eventuate-local-java-cdc-connector-mysql-binlog/src/main/java/io/eventuate/local/mysql/binlog/MySqlBinaryLogClient.java:204]: This should be debug logging

**Review by cer — CHANGES_REQUESTED** [review]

File: eventuate-local-java-cdc-connector-mysql-binlog/src/main/java/io/eventuate/local/mysql/binlog/MySqlBinaryLogClient.java (hunk 2 above)
Comment (dartartem) [reply eventuate-local-java-cdc-connector-mysql-binlog/src/main/java/io/eventuate/local/mysql/binlog/MySqlBinaryLogClient.java:204]: ok

File: eventuate-local-java-cdc-connector-db-log-common/src/main/java/io/eventuate/local/db/log/common/DatabaseOffsetKafkaStore.java (hunk 1 above)
Comment (dartartem) [reply eventuate-local-java-cdc-connector-db-log-common/src/main/java/io/eventuate/local/db/log/common/DatabaseOffsetKafkaStore.java:46]: ok

**Pull request merged**